_categories/sensors.md

```diff
@@ -29,10 +29,10 @@ subcategories:
   
   - title: "Sensors integrated with ROS (gazebo_ros_pkgs)"
     items: 
-      - title: 'Differences between the gazebo_ros_pkgs sensors and gazebo native sensors'
-        url: 'http://answers.gazebosim.org/question/TODO' # check
-        desc: 'How are the gazebo_ros_pkgs sensors different from the native Gazebo sensors and why is recommended to use them in ROS'
-        star: true
+      # - title: 'Differences between the gazebo_ros_pkgs sensors and gazebo native sensors'
+      #   url: 'http://answers.gazebosim.org/question/TODO' # check
```

Comment:
not very detailed but it can help this file as link here: https://github.com/ros-simulation/gazebo_ros_pkgs/blob/kinetic-devel/SENSORS.md